fission/environment.go

```diff
@@ -218,6 +218,10 @@ func envUpdate(c *cli.Context) error {
 
 	env.Spec.AllowAccessToExternalNetwork = envExternalNetwork
 
+	if c.IsSet("mincpu") || c.IsSet("maxcpu") || c.IsSet("minmemory") || c.IsSet("maxmemory") || c.IsSet("minscale") || c.IsSet("maxscale") {
+		log.Fatal("Currently it's not support to update resources limits/requests after creating the environment, please recreate it instead.")
```

Comment:
1. No logs in the CLI -- it should simply be a printf and exit.

2. "Updating resource limits/requests for existing environments is currently unsupported; re-create the environment instead.  (Issue NNN)".

And let's file an issue to track allowing these updates.

Reply:
1. It's not the built-in log package, it prints message and exit. (fission/fission/log)
issue to track resource change to env: https://github.com/fission/fission/issues/972